easybuild/easyconfigs/o/OrfM/OrfM-0.6.1-foss-2016b.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     LGPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'OrfM'
+version = '0.6.1'
+
+homepage = 'https://github.com/wwood/OrfM'
+description = """A simple and not slow open reading frame (ORF) caller. 
+No bells or whistles like frameshift detection, 
+just a straightforward goal of returning a FASTA file 
+of open reading frames over a certain length from a FASTA/Q file 
+of nucleotide sequences.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+sources = ['v%(version)s/%(namelower)s-%(version)s.tar.gz']
```

Comment:
@boegel any particular reason for this? I thought sources just append address after source_url.